### ISSUE TITLE
GO term improvements 

### ISSUE THREAD

**Issue by AntonPetrov** [Rfam/rfam-production#13]
- [ ] develop QC for checking that all families of a certain type have certain GO terms
- [ ] develop scripts for bulk editing GO terms so that it is possible to remove, add, or replace GO terms  in all families

**Comment by kalvari** [comment]
Not sure yet, but perhaps QC should be done in a separate step. At least what I currently have in mind. Curator can run QC to identify any issues and/or families of a specific type that should have GO terms but don't, and then add new entries to the spreadsheet. Alternatively, we can have a "_post_process" like table, that will only pick up the families that need to be modified and work on those only.

**Comment by AntonPetrov** [comment]
I agree, QC _should_ be a separate step, perhaps a script that run by a curator and/or by cron. Just to clarify - QC and bulk editing are independent tasks.

**Comment by kalvari** [comment]
Partially done. replace and remove is feasible from go validation file. Need to implement "ADD" functionality and enable these from curation (.tsv) files
